minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneAccumuloCluster.java

```diff
@@ -177,6 +187,17 @@ public void stop() throws IOException {
     for (ServerType type : ALL_SERVER_TYPES) {
       control.stopAllServers(type);
     }
+    clusterState = State.STOPPED;
+  }
+
+  @Override
+  public void terminate() throws Exception {
+    Preconditions.checkState(clusterState == State.STOPPED,
+        "Cannot terminate a cluster that is not stopped.");
```

Comment:
Would it be useful to have terminate imply stopped?

Reply:
Implemented in b5a3214 and javadoc updated.